src/dbnode/integration/integration_data_verify.go

```diff
@@ -73,7 +74,7 @@ func verifySeriesMapForRange(
 	input generate.SeriesBlock,
 	expectedDebugFilePath string,
 	actualDebugFilePath string,
-) {
+) bool {
```

Comment:
Hm this is probably a good change.  Especially when called in an async goroutine because otherwise the goroutine just dies when a `require.Foo(...)` fails.  👍

Reply:
yeah plus gopter require you to return (false, error) when something goes wrong. If you just panic it can't really do its thing